### PR TITLE
feat(devices): add uid based roll out instead of metrics uuid

### PR DESCRIPTION
**Pull request by vladikoff**
@rfk r?

### CODE REVIEW

File: clients/devices.js [hunk 1]
```diff
@@ -3,13 +3,19 @@
...
+    var sampleRate = 0.1;
+
     if (subject) {
-      if (subject.isMetricsEnabledValue || subject.forceDeviceList) {
+      if (subject.forceDeviceList) {
         return true;
       }
+
+      if (subject.uid) {
+        return !! (this.bernoulliTrial(sampleRate, subject.uid));
```
Comment (vladikoff) [inline clients/devices.js:17]: this is based on https://github.com/mozilla/fxa-content-experiments/blob/dev/metrics/is-sampled-user.js#L9

**Comment by rfk** [comment]
LGTM, r+

**Comment by rfk** [comment]
Does the uid get passed in automagically or do we also need a content-server change to send it?

**Comment by vladikoff** [comment]
> do we also need a content-server change to send it?

Yeap also will need a change

**Pull request merged**